### DISCUSSION TITLE
Modifying the default font size

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1230] [Ideas] [answered]
I feel that the default font size of `1.5x` makes it seem like there is a lot of extra screen real estate that is not really being taken up by anything. I believe that the font size of `2x`makes the text fit on the screen perfectly. Share your thoughts on this.
`1.5x`
![1 5x](https://user-images.githubusercontent.com/79121360/114287682-92a5ca80-9a1d-11eb-8b66-6a3a91c16c7f.png)

`2x`
![2x](https://user-images.githubusercontent.com/79121360/114287688-9fc2b980-9a1d-11eb-8e2b-bda8d5e36cf9.png)

**Comment by unrealapex** [comment, marked as answer]
Well, never mind.

**Reply by frogfrogfrogfrogfrog** [reply]
Well it seems like a neat idea. Changing the default font size so that the default text fits perfectly. I will also say that the default font size seems quite small so changing it to be bigger would be nice for people who are just getting onto Monkeytype that don't know how to change the font size.

**Reply by Miodec** [reply]
I didnt fully understand this idea as it seemed like it was based on the width of the text. Like, if the default custom text was longer it would fit it nicely without changing the size. Plus, custom mode is not the main way people are using the website, or the first mode that they see.

**Reply by unrealapex** [reply]
By screen real estate, I mean in terms of the height of the text, not the width it takes up. The blank areas are above and below the text.  I used custom mode as an example because when changing the font size be it through command line or on the settings page, the typing text changes. But I do think that `1.5x` does allow you to looks several words ahead giving you an advantage.
